examples/channels-for-correlation/channels_correlation_service.out

```diff
@@ -0,0 +1,5 @@
+# To start the service, navigate to the directory that contains the
+# `.bal` file and use the `ballerina run` command.
+$ ballerina run channels.bal
+ballerina: initiating service(s) in 'channels.bal'
```

Comment:
Could you please re-check the output against the current master? We've removed the "ballerina: " with the initiating service(s) message, and it should be `Initiating service(s) in 'channels.bal'` now.